libs/ts-rest/core/src/lib/dsl.ts

```diff
@@ -128,9 +131,10 @@ export type AppRouter = {
   [key: string]: AppRouter | AppRoute;
 };
 
-export type RouterOptions = {
+export type RouterOptions<TPrefix extends string> = {
```

Comment:
I think we can default this to `''` so we don't have to explicitly pass `<string>` everywhere.

```suggestion
export type RouterOptions<TPrefix extends string = ''> = {
```

Reply:
Actually I might be slightly confused, it's late lol but I think the correct way would be `TPrefix extends string = string`

Reply:
@Jwcode-uk I pulled your code locally just to make sure I wasn't just spewing fairy dust, and it indeed worked, so I pushed the changes. I hope you don't mind! 🙏

Reply:
Appreciate the help!!